Makefile

```diff
@@ -25,7 +25,11 @@ test-unit:
 
 test-unit-with-coverage:
 	$(call print_msg, Running unit tests with coverage... )
-	nosetests -v --with-xcoverage --cover-package=app test/unit test/integration
+	nosetests -v --with-xcoverage --cover-package=app test/unit
+
+test-integration-with-coverage:
+	$(call print_msg, Running unit tests with coverage... )
+	nosetests -v --with-xcoverage --cover-package=app test/integration
```

Comment:
Sounds good. I thought this would mess with coveralls but it doesn't look like coverage went down at all so it must be fine.